common/src/main/scala/models/elections/ElectionResults.scala

```diff
@@ -0,0 +1,25 @@
+package models.elections
+
+import java.net.URI
+
+import models.JsonUtils
+import play.api.libs.json.Json
+
+case class CandidateResults(
+  name: String,
+  states: List[String],
+  electoralVotes: Int,
```

Comment:
Should these be `Long` instead? _(or am I misunderstanding the US election system ?)_